back-dispatcher/routers/TopHeadlinesRoute/topHeadlinesModel.ts

```diff
@@ -0,0 +1,7 @@
+import mongoose from "mongoose";
+import { topheadlinesSchema } from "./topHeadlinesSchema";
+
+export const topHeadlineModel = mongoose.model(
+  "topHeadlineArticle",
+  topheadlinesSchema
+);
```

Comment:
put this code in the schema ts file.
the model ts file is handling  the functionality for example:
find, delete, ext.